steps/reasons-for-appealing/evidence-provide/EvidenceProvide.js

```diff
@@ -8,12 +8,21 @@ const Joi = require('joi');
 const { titleise } = require('utils/stringUtils');
 const userAnswer = require('utils/answer');
 const i18next = require('i18next');
+const { isIba } = require('utils/benefitTypeUtils');
 
 class EvidenceProvide extends SaveToDraftStore {
   static get path() {
     return paths.reasonsForAppealing.evidenceProvide;
   }
 
+  /** TODO: Welsh translation for RDN and sentance of welsh MRN */
+  get noticeType() {
```

Comment:
should this not be in a content file?

Reply:
@benouaer  This is used within the content file to dynamically render the noticeType. Used this method for single text changes